tests/performance/specs/single-post.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { test } from '@wordpress/e2e-test-utils-playwright';
+
+/**
+ * Internal dependencies
+ */
+import { camelCaseDashes } from '../utils';
+
+const results = {
+	timeToFirstByte: [],
+	largestContentfulPaint: [],
+	lcpMinusTtfb: [],
+};
+
+const themes = [ 'twentytwentyone', 'twentytwentythree', 'twentytwentyfour', 'twentytwentyfive' ];
+
+const locales = [ 'en_US', 'de_DE' ];
```

Comment:
Obviously there's a lot of duplicate code across these files that we should probably consolidate at some point, but I think an easy win would be to at least consolidate these values, which are essentially configuration parameters for our performance monitoring.

Could you maybe create a `config.js` file that exports the `themes` and `locales` lists, so that each of the spec file can import them rather than duplicating the lists? I think that would already be a great improvement and quick enough to implement as part of this.